apps/portal/src/app/features/transcript-analyzer/transcript-analyzer.component.scss

```diff
@@ -0,0 +1,25 @@
+@import 'primeflex/primeflex.scss';
```

Comment:
I don't remember but do we have to import it in every scss file?

Reply:
without importing it I am not able to use styleclass in scss file and ng server is throwing error, I tried to import it in global scss but still I am not able to use styleclass in component.scss